docs/source/api.yaml

```diff
@@ -445,6 +446,11 @@ definitions:
       createValueOnlyIndices:
         type: boolean
         example: true
+        description: 'If true, enable search for any term, without specifying a field name'
```

Comment:
Just to be sure - you made these changes  in the open-api directory  then ran npm run build to get the changes in this api.yaml file correct?